tests/e2e/specs/modules/analytics/setup-gcp-no-account-no-tag.test.js

```diff
@@ -184,14 +192,19 @@ describe( 'setting up the Analytics module using GCP auth with no existing accou
 			'.googlesitekit-analytics__select-account .mdc-select__selected-text',
 			{ text: '' }
 		);
-		await expect( page ).toMatchElement(
-			'.googlesitekit-analytics__select-property .mdc-select__selected-text',
-			{ text: '' }
-		);
-		await expect( page ).toMatchElement(
-			'.googlesitekit-analytics__select-profile .mdc-select__selected-text',
-			{ text: '' }
-		);
+
+		await expect( page ).toClick( '.mdc-select', {
+			text: '',
+		} );
+
+		await Promise.all( [
+			expect( page ).toClick( '.mdc-menu-surface--open .mdc-list-item', {
+				text: /test account a/i,
+			} ),
+			page.waitForResponse( ( res ) =>
+				res.url().match( 'modules/analytics-4/data' )
+			),
```

Comment:
We probably need to be more specific about which response we're waiting for here since this would match anything for GA4. We didn't wait for a response here before either, so I'd question whether this is really necessary. We also wait for 1s right after this which should be sufficient for a wait here.

Reply:
It was to double ensure that all needed ga4 requests made through (properties, webdatastreams-batch). But yeah, 1s should be enough.